### ISSUE TITLE
Feature request: Redis backend for Google DB ?

### ISSUE THREAD

**Issue by JoeOrtiz** [cesanta/docker_auth#126]
Instead of having a file based store for the google auth token, I believe it would be best to keep it as a redis backend, that way it can be distributed. This way we can launch multiple docker_auth's for redundancy.

**Comment by rojer** [comment]
makes sense. we have a [TokenDB interface](https://github.com/cesanta/docker_auth/blob/master/auth_server/authn/tokendb.go#L38) now, so it should be easy enough to implement.
i don't have experience with Redis or time to implement it though, so all i can say is - PRs welcome :)
